features/yield-calculator/YieldCalculator.tsx

```diff
@@ -139,12 +139,21 @@ const YieldCalculator = () => {
                 />
               </FormInput>
             </Grid>
-            <Grid item xs={4}>
-              <Box pt={4}>
-                <Typography variant="h6">
-                  Yearly APR: {prettyPercentage(yieldAmount)}%
-                </Typography>
-              </Box>
+            <Grid item md={4} xs={12}>
+              <Hidden only={["sm", "xs"]}>
+                <Box pt={4}>
+                  <Typography variant="h6">
+                    Yearly APR: {prettyPercentage(yieldAmount)}%
```

Comment:
You need to merge in the latest `develop` changes that affects this YieldCalculator.tsx file